src/RDBViewProviders.ts

```diff
@@ -1,19 +1,25 @@
 import * as vscode from 'vscode';
+import * as path from 'path';
 import { OnDeviceComponent } from 'roku-test-automation';
 
 export class RDBRegistryViewProvider implements vscode.WebviewViewProvider {
     private view?: vscode.WebviewView;
     private odc?: OnDeviceComponent;
+    private extensionPath: string;
 
-    constructor(context: vscode.ExtensionContext, private rdbOutputChannel: vscode.OutputChannel) {}
+    constructor(context: vscode.ExtensionContext, private rdbOutputChannel: vscode.OutputChannel) {
+        this.extensionPath = context.extensionPath + "/dist/ui/rdb"
```

Comment:
Feel like it makes sense to name this something different for clarification. Maybe this.rdbBasePath